conda_build/skeletons/cran.py

```diff
@@ -966,11 +951,12 @@ def skeletonize(in_packages, output_dir=".", output_suffix="", add_maintainer=No
                     archive_details['cranurl'] = archive_details['binaries'][package][-1][1]
                     archive_details['conda_version'] = archive_details['binaries'][package][-1][0]
                     archive_details['cran_version'] = archive_details['conda_version'].replace('_', '-')
-                    avaliable_artefact = True
+                    available_artifact = True
```

Comment:
https://writingexplained.org/artefact-vs-artifact-difference

Reply:
Ah! That was muscle memory. My original intent was to correct the misspelling of "available"